vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -235,12 +235,23 @@ void LedgerImpl::OnPostData(
     return;
   }
 
-  std::vector<std::map<std::string, std::string>> twitchParts;
-  if (TWITCH_MEDIA_TYPE == type) {
+  if (type == TWITCH_MEDIA_TYPE) {
+    std::vector<std::map<std::string, std::string>> twitchParts;
     braveledger_media::GetTwitchParts(post_data, &twitchParts);
     for (size_t i = 0; i < twitchParts.size(); i++) {
       bat_get_media_->ProcessMedia(twitchParts[i], type, std::move(visit_data));
     }
+    return;
+  }
+
+  if (type == VIMEO_MEDIA_TYPE) {
+    std::vector<std::map<std::string, std::string>> parts;
```

Comment:
Good place to use `auto` here, if possible.

Reply:
same as above